AnnoyancesFilter/Other/sections/self-promo.txt

```diff
@@ -1473,6 +1473,7 @@ theguardian.com##.contributions__adblock
 ! SECTION: Self-promo - Regular rules
 !
 detail.chiebukuro.yahoo.co.jp###v_bnr
+itmedia.co.jp##.colBoxITT
```

Comment:
Are they linked to their own group sites? I meant if they lead to third-party sites e.g. `https://www.pref.yamaguchi.lg.jp/soshiki/32/101159.html`, you can add `itmedia.co.jp##ITT` to Japanese filter.

Reply:
The two at the top of the site go to third-party site.
<details><summary>screenshot - 1</summary>

![](https://i.postimg.cc/sXsNhV8Q/image.png)

</details><br/>

But `##.colBoxITT` on the sidebar of the site go to own group site.

<details><summary>screenshot - 2</summary>

![](https://i.ibb.co/zm36pqQ/image.png)

</details><br/>

Reply:
Maybe
`###ITT` => JPF
`###RANKTXT` => Annoyances

Reply:
Please also add `###cmsSubPR` to JPF seen in `https://www.itmedia.co.jp/business/articles/2209/13/news008.html`

Reply:
I see. I will change the rule